source/containers.tex

```diff
@@ -6716,7 +6716,7 @@
 \begin{itemdescr}
 \pnum
 \expects
-\tcode{T} is \oldconcept{DefaultInsertable} into \tcode{*this}.
+\tcode{T} is \oldconcept{DefaultInsertable} into \tcode{deque}.
```

Comment:
Should I change `T` to `value_type` in each case as well?

Reply:
I don't think that's necessary (and would require changes in places that this PR doesn't already touch).